include/openenclave/internal/eeid.h

```diff
@@ -15,6 +15,13 @@
 
 #include <openenclave/bits/eeid.h>
 
+/* When signing EEID base images we don't know the size that the final image
+ * will have, so we chose a reasonably large size here (64GB). */
+#define EEID_ELRANGE 0x1000000000
```

Comment:
This number was previously in an SGX-only file and eeid.h is not in an SGX directory or with an sgx in the filename.  So this needs to have SGX in the macro name like lines 22 and 23 do.  Suggest OE_SGX_EEID_ELRANGE

Reply:
Good point, I'll add that!